cylc/flow/workflow_files.py

```diff
@@ -871,13 +871,14 @@ def get_symlink_dirs(reg: str, run_dir: Union[Path, str]) -> Dict[str, Path]:
             target = path.resolve()
             if target.exists() and not target.is_dir():
                 raise WorkflowFilesError(
-                    f'Invalid Cylc symlink directory {path} -> {target}\n'
-                    f'Target is not a directory')
+                    f'Invalid symlink at {path}.\n'
+                    f'Link target is not a directory: {target}')
             expected_end = str(Path('cylc-run', reg, _dir))
             if not str(target).endswith(expected_end):
                 raise WorkflowFilesError(
-                    f'Invalid Cylc symlink directory {path} -> {target}\n'
-                    f'Expected target to end with "{expected_end}"')
+                    f'Invalid symlink at {path}\n'
+                    f'Target should end with "{expected_end}"'
+                    f'but was {target}.')
```

Comment:
I think this `but was` sounds a bit weird, What about replacing the middle and last line with: 
`f'{target} should end with "{expected_end}"'`
Feel free to dismiss, it isn't a blocker to merge.

Reply:
```suggestion
```